src/platforms/javascript/common/errors/breadcrumbs/index.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Breadcrumbs
+excerpt: ""
+---
+
+Sentry supports a concept called Breadcrumbs, which is a trail of events that happened prior to an issue. Oftentimes these events are very similar to traditional logs, but also have the ability to record more rich structured data.
+
+## What Sentry Sends Automatically
+
+SDKs will automatically start recording breadcrumbs by enabling integrations, such as `Sentry.Integrations.Breadcrumbs`. Learn more in our documentation for [Default Integrations](../integrations/default/). In addition, the JavaScript SDK will automatically record all location changes and automatically records certain events, such as changes to the URL and XHR requests to add context to an error.
+
+Each crumb in the trail includes:
```

Comment:
Why do we describe the schema here, we already describe it in develop.sentry.dev? This is in the namespace for the JS SDK so I would only expect the SDK-specific infromation